### ISSUE TITLE
Pytorch 1.6.0 seem to leak memory in conv2d

### ISSUE THREAD

**Issue by Wang-zipeng** [openvinotoolkit/nncf_pytorch#141]
I'm using Pytorch 1.6.0, and there is only one conv, my codes are as the follows:
`inputs = torch.randn(1, 3, 512, 512).cuda()`
`conv = torch.nn.Conv2d(3, 64, (7, 7), stride=2, padding=3, bias=False).cuda()`
`output = conv(inputs)`
Before execute the conv operation, the GPU memory usage from the nvidia-smi is 1019MB, after the conv operation the GPU memory usage is 1429, and the conv operation consume about 410MB, I know the im2col may consume a huge memory when input size is large. What i can't understand is that after the conv operation the GPU usage not become low. I think there is maybe memory leak in conv2d or there is something wrong in my experiment?

**Comment by Wang-zipeng** [comment]
repo wrong
